mlserver/registry.py

```diff
@@ -155,6 +156,7 @@ async def _load_model(self, model: MLModel):
             # appears as a not-ready (i.e. loading) model
             self._register(model)
 
+            sys_tracer.tp_model_load_begin(model.name, model.version)
```

Comment:
Thinking out loud here as well - it looks like this would fit really well as a context manager (i.e. as `with sys_tracer.tp_model_load(): .....`)

Reply:
Cool idea! This could be quite useful for tracepoints that are often fired in pairs within the same function. 

Sometimes the "begin" tracepoint will be fired in a different function than the "end", so we'll need to retain the ability to fire them individually.